recipe/meta.yaml

```diff
@@ -102,10 +102,11 @@ test:
       # Test with the two currently supported lapack implementatons
       # One test done on different versions of python on each platform
       - liblapack * *openblas         # [py==36]
-      - liblapack * *mkl              # [py==37]
+      - liblapack * *mkl              # [py==37 and not (aarch64 or ppc64le)]
+      - liblapack * *openblas         # [py==37 and (aarch64 or ppc64le)]
```

Comment:
I think you can simply drop this, and it will pull out the default version of blas automatically.